gravitee-apim-console-webui/src/index.scss

```diff
@@ -87,6 +85,25 @@ body {
   @include material-layout-compatibility();
 
   @import './scss/gio-global-style';
+
+  .gio-root {
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
```

Comment:
We need to find something else. i have tested and it breaks the scroll in pages :sad: